README.md

```diff
@@ -6,7 +6,7 @@ and other changes required for customizing Web Flow for clients.
 
 ## Documentation
 
-For the most recent documentation and tutorials, please visit [PowerAuth Web Flow Customization Documentation](./docs/Home.md).
+For the most recent documentation and tutorials, please visit [PowerAuth Web Flow Customization Documentation](https://developers.wultra.com/docs/latest/powerauth-webflow-customization/).
```

Comment:
This will not work. We need to get rid off that "/latest/" part in the link, due to broken case sensitive renames in other respos. I would keep the file as is.